modules/utils/docker/entrypoint.sh

```diff
@@ -1,57 +1,157 @@
-#! /bin/bash
+#! /bin/bash -x
 
 COMMS_PCB_VERSION_FILE="/opt/hardware/comms_pcb_version"
 
 calculate_wifi_channel()
 {
-    # arguments:
-    # $1 = wifi frequency
-    # return values: retval_band, retval_channel as global
-    # Set 2.4/5GHz frequency band and channel
-
-    if [ "$1" -ge  "5160" ] && [ "$1" -le  "5885" ]; then
-        retval_band="a"
-        retval_channel=$((("$1"-5000)/5))
-    elif [ "$1" -ge  "2412" ] && [ "$1" -le  "2472" ]; then
-        retval_band="g"
-        retval_channel=$((("$1"-2407)/5))
-    else
-        echo "ERROR! frequency out of range!"
-        exit 1
-    fi
+  # arguments:
+  # $1 = wifi frequency
+  # return values: retval_band, retval_channel as global
+  # Set 2.4/5GHz frequency band and channel
+
+  if [ "$1" -ge 5160 ] && [ "$1" -le 5885 ]; then
+    retval_band="a"
+    retval_channel=$((("$1"-5000)/5))
+  elif [ "$1" -ge 2412 ] && [ "$1" -le 2472 ]; then
+    retval_band="g"
+    retval_channel=$((("$1"-2407)/5))
+  else
+    echo "ERROR! frequency out of range!"
+    exit 1
+  fi
+}
+
+create_ap_config()
```

Comment:
Moved to a new place unecessarily. If you look at the original file it was originally down below.